projects/aca-shared/src/lib/components/tool-bar/toolbar-button/toolbar-button.component.html

```diff
@@ -16,6 +16,7 @@
     <button
       [id]="actionRef.id"
       [color]="data?.color || color"
+      [ngClass]="{'secondary-button-color': !data?.color}"
```

Comment:
you set the "primary" color in the app config, but this is not reflected as you just force a static background